docs/sources/tempo/configuration/usage-tracker.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Usage tracker
+description: Learn how to configure the usage tracker for cost attribution.
+weight: 700
+aliases:
+- /docs/tempo/configuration/usage-tracker   
+---
+
+# Usage tracker
+
+The usage tracker is a feature that accurately tracks the amount of ingested traffic using a set of custom labels on a per-tenant basis, providing fine-grained control over your data.
+
+This feature has to be enabled first in the [distributor](https://grafana.com/docs/tempo/<TEMPO_VERSION>/configuration/#distributor):
+
+```
+ usage:
+        cost_attribution:
+            # Enables the "cost-attribution" usage tracker. Per-tenant attributes are configured in overrides.
+            [enabled: <boolean> | default = false]
```

Comment:
What do you think about moving the example above to the introduction and changing this procedural example to be more like what someone would include in their actual config? This example is from lines 193-196 in `manifest.md`: 

```
distributor:
    usage:
        cost_attribution:
            max_cardinality: 10000
            stale_duration: 15m0s
```

Reply:
If you choose to use the above example, then we should explain what the configuration options impact.

Reply:
I have used a more concrete example. I think using all the options is not needed. We can left them with their default values

Reply:
Okay, I think the easiest way to get around this is to include the entire configuration example from the configuration.md file. That way people have the options and the explanation. I'll create a suggestion doing that.